invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/oai-details.js

```diff
@@ -0,0 +1,46 @@
+// This file is part of InvenioRdmRecords
+// Copyright (C) 2022 CERN.
+//
+// Invenio RDM is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import React from "react";
+import ReactDOM from "react-dom";
+import _get from "lodash/get";
+import LinksTable from "./LinksTable";
+import AdminDetailsView from "@js/invenio_administration/details/AdminDetailsView.js";
+import { OverridableContext } from "react-overridable";
+import _cloneDeep from 'lodash/cloneDeep';
+
+const mainDomContainer = document.getElementById("invenio-details-config");
+const fields = JSON.parse(mainDomContainer.dataset.fields);
+const apiEndpoint = _get(mainDomContainer.dataset, "apiEndpoint");
+const pidValue = JSON.parse(mainDomContainer.dataset.pid);
+const title = mainDomContainer.dataset.title;
+const actions = JSON.parse(mainDomContainer.dataset.actions);
+const displayEdit = JSON.parse(mainDomContainer.dataset.displayEdit);
+const displayDelete = JSON.parse(mainDomContainer.dataset.displayDelete);
+
+// const overriddenComponents = {
```

Comment:
The overridable part can be removed. It was just here to showcase how to override the whole component. 

In OAI-PMH I don't think we want to override the whole component, since we still want the main table to be displayed. Therefore, composition was added to `AdminDetailsView`.